src/jumpstart/saga.ts

```diff
@@ -191,3 +203,129 @@ export function* dispatchPendingERC721Transactions(
     }
   }
 }
+
+export function* sendJumpstartTransactions(
+  action: PayloadAction<JumpstartTransactionStartedAction>
+) {
+  const { serializablePreparedTransactions, sendToken, sendAmount } = action.payload
+
+  try {
+    const networkId = sendToken.networkId
+    const jumpstartContractAddress = getDynamicConfigParams(
+      DynamicConfigs[StatsigDynamicConfigs.WALLET_JUMPSTART_CONFIG]
+    ).jumpstartContracts?.[networkId]?.contractAddress
+    if (!jumpstartContractAddress) {
+      throw new Error(
+        `Jumpstart contract address for send token ${sendToken.tokenId} on network ${networkId} is not provided in dynamic config`
+      )
+    }
+
+    const createStandbyTxHandlers = []
+    const preparedTransactions = getPreparedTransactions(serializablePreparedTransactions)
+
+    // in this flow, there should only be 1 or 2 transactions. if there are 2
+    // transactions, the first one should be an approval.
```

Comment:
i had that before, and removed it - https://github.com/valora-inc/wallet/pull/5091#discussion_r1524621485 essentially i don't think that it's necessary to be so overly defensive in this saga because we are creating the transactions ourselves. if we do want to decode the transactions, we should do it both for the approval and the actual transaction in order to populate the fields for the pending transactions. however then we'd also need to handle errors / decide what to do when the transactions are _not_ what we expect, which seems like a lot of extra unnecessary thinking if the transactions were created by us in the first place....

Reply:
happy to discuss this further, i can be persuaded to bring back the transaction decoding but i'll keep it simple and merge as is for now to unblock the other PRs relating to this